<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShiftRemovalDialog</class>
 <widget class="QDialog" name="ShiftRemovalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shift Removal Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="CancellButton">
     <property name="text">
      <string>Cancell</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="WorkersShiftsComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="AcceptButton">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="TitleLabel">
     <property name="text">
      <string>Worker's Shift's</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>buttonofAcceptWasClicked()</slot>
  <slot>buttonofCancellWasClicked()</slot>
 </slots>
</ui>
